nntrainer/cl_context.cpp

```diff
@@ -129,6 +129,7 @@ void ClContext::initBlasClKernels() {
   }
 
   registerClKernel(sgemv_cl_kernel_, "sgemv_cl");
+  registerClKernel(sgemv_cl_noTrans_kernel_, "sgemv_cl_noTrans");
```

Comment:
Quick Question) 

in registerClKernel function 
```.cpp
  // check if created before
  if (ocl_kernel_map.find(kernel_name) != ocl_kernel_map.end()) {
    ml_logi("Kernel already registered and initialized: %s",
            kernel_name.c_str());
    return ocl_kernel_map[kernel_name];
  }
```
It appears to check whether it is already registered or not.

but at line 134 & line 132 

```.cpp
registerClKernel(sgemv_cl_noTrans_kernel_, "sgemv_cl_noTrans");
```
It appears that these two lines may have been duplicated inadvertently or intentionally. 
is there any specific purpose behind this repetition, or was it simply a mistake?

Reply:
There is a subtle difference in line 132 and 134. One is `sgemv` and the other is `sgemm`:
Line 132: `registerClKernel(sgemv_cl_noTrans_kernel_, "sgemv_cl_noTrans");`
Line 134: `registerClKernel(sgemm_cl_noTrans_kernel_, "sgemm_cl_noTrans");`

Reply:
Oh, I didn't see it's detail. Thank you for your explanation.